src/azfswatchdog

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash 
+
+#  
+# How often does the watchdog look for unmounts and/or IP address changes for 
+# Blob endpoints. 
+# 
+
+MONITOR_INTERVAL_SECS=5 
+
+# 
+# This stores the map of local IP and share name and external blob endpoint IP. 
+# 
+
+IPMAP="/tmp/azfs-ipmap.txt"
+
+
+# 
+# Watchdog for monitoring unmounts and more importantly change in blob endpoint 
+# addresses possibly as a result of migration. 
+# 
+# TODO: This must be a separate process started by init/systemd. 
+# 
+
+azfs_watchdog() 
+{ 
+    # Disable signal to make this slightly more resilient. 
+    trap "" INT QUIT TSTP 
+
```

Comment:
before starting the loop where it monitors MOUNTMAP for changes, it should first create the dir and file if it doesn't exist.

Reply:
Added.